### ISSUE TITLE
Why are there two different orientations on the screenshot

### ISSUE THREAD

**Issue by zhamilyaa** [yayaneath/GeoGrasp#11]
![image](https://user-images.githubusercontent.com/40009146/163362995-76c1d12f-7eb0-48d7-b035-ba72d141b6d5.png)

I have a concern regarding this screenshot. There are two different orientations for grasping on minion and soda can.

**Comment by yayaneath** [comment]
Good point. The orientation on the minion is different because the IK of our robot (multifingered hand) could not handle well the grasp so the points were reversed. For a two-fingered gripper the pose should be fine in either direction
